src/Person/PersonRow.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { useHistory, useParams } from 'react-router-dom';
+import cn from 'classnames';
+import './Person.scss';
+
+type Props = {
+  people: Person[];
+};
+
+export const PersonRow: React.FC<Props> = ({ people }) => {
+  const history = useHistory();
+  const { personSlug } = useParams();
+
+
+  const handleChange = (personUrl: string) => {
+    history.push({
+      pathname: `/people/${personUrl}`,
+    });
+  };
+
+  return (
+    <>
+      {people.map((person: Person) => (
+        <tr
+          className={cn({
+            person: true,
+            person__active: person.slug === personSlug,
+          })}
```

Comment:
```suggestion
          className={cn('person', {
            person__active: person.slug === personSlug,
          })}
```
if you always have some class